libraries/botframework-schema/src/index.ts

```diff
@@ -2148,3 +2148,28 @@ export interface IStatusCodeError {
     statusCode: StatusCodes;
     message?: string;
 }
+
+export interface AdaptiveCardAuthentication {
+    id: string;
+    connectionName: string;
+    token: string;
+}
+
+export interface AdaptiveCardInvokeAction {
+    type: string;
+    id: string;
+    verb: string;
+    data: Record<string, unknown>;
+}
+
+export interface AdaptiveCardInvokeResponse {
+    statusCode: number;
+    type: string;
+    value: Record<string, unknown>;
+}
+
+export interface AdaptiveCardInvokeValue {
+    action: AdaptiveCardInvokeAction;
+    authentication: AdaptiveCardAuthentication;
+    state: string;
+}
```

Comment:
Docstrings? Are all of the members correctly noted as required and are the values for the `type` members open-ended?

Reply:
I believe these interfaces are all faithful ports of https://github.com/microsoft/botbuilder-dotnet/pull/5227/files#diff-a00af63f48728bbd7aa5309b5367372793b0a6ec8e5deac9dd3ae78f592f90d4, I'm adding the docstrings though.